### PR TITLE
fix(atom/button): use prop rel in AtomButton when it is a link

### PR DESCRIPTION
**Pull request by rmoralp**
`rel` attribute can have different values as you can see in: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel#:~:text=The%20rel%20attribute%20defines%20the,which%20the%20attribute%20is%20found.

We need to enable the possibility of sent the value we want in props and use it. `nofollow`, `noopener`, `noreferrer` are common values`rel` could countains

### CODE REVIEW

**Review by adriadoz — APPROVED** [review]

File: components/atom/button/src/Button.js [hunk 1]
```diff
@@ -14,13 +14,12 @@ const Button = ({
 }) => {
   if (isSubmit) attrs.type = 'submit'
   if (isButton) attrs.type = 'button'
+
+  const defaultRel = target === '_blank' ? 'noopener' : undefined
+  const rel = attrs.rel || defaultRel
```
Comment (jordevo) [inline components/atom/button/src/Button.js:19]: wouldn't this achieve the same? 

```suggestion
  const {rel = defaultRel} = attrs
```

Comment (tomcask) [reply components/atom/button/src/Button.js:19]: I like the more traditional way

image

vs

image

Your suggestion does not provide more readability and generates more lines of code.

**Review by tomcask — APPROVED** [review]

**Review by javiauso — APPROVED** [review]

File: components/atom/button/src/Button.js (hunk 1 above)
Comment (jordevo) [reply components/atom/button/src/Button.js:19]: gotcha... I'm just so used to assigning while destructuring...

**Review by jordevo — APPROVED** [review]

**Review by midudev — APPROVED** [review]

**Comment by midudev** [comment]
I've merged master in order to get tests to pass. ✅

**Pull request merged**